packages/v4/src/content/design-guidelines/components/time picker/time-picker.md

```diff
@@ -0,0 +1,104 @@
+---
+id: Time picker
+section: components
+related: ["Calendar month", "Date picker"]
+---
+
+A **time picker** allows users to select a time from a list of options. Time can be displayed in hours and minutes, but not in seconds.
+
+## Elements
+
+<img src="./img/Time picker- elements.png" alt="Time picker elements" width="362"/>
+
+1. **Time field**: Displays a user’s selected time. Always fill the time field with a placeholder time format (HH:MM) or a pre-selected time before users select their time by entering the field, typing ahead, scrolling, and selecting their desired time from the menu. The time field should never be empty.
+2. **Time field in focused state**: Opens a menu list of time options. 
+3. **Menu**: Contains a list of times in set increments, such as 12:00 AM to 11:30 PM in 30 minute increments.
+4. **Menu item in hovered state**
+5. **Menu item in disabled state**
+
+## Usage
+Use time pickers to enable users to select or input a time for use cases like item filtration or scheduling in a form.
+
+### Single time selection
+A time picker allows users to select a single time. When a user enters a new time in the time field, the new time automatically replaces any placeholder or previously-selected value in that input field.
+
+Typically, time picker menus list selectable times in 30-minute increments. However, users can choose to forgo typeahead and menu selection by directly inputting desired times that don’t fall on 30-minute increments, such as 12:38 PM. 
```

Comment:
Re-word "However, users always have the option to input any desired times that don't fall on the suggested time increments. For example, a user could type in 12:38 PM despite the list of options only showing 12:30 PM and 1:00 PM options."